software/bringup-scripts/pic_hdmi.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-./../power_init.sh
+./../scripts/power_init.sh
```

Comment:
the `./` part is not needed, only `../scripts/power_init.sh` is sufficient

(this applies to all the changes not just this one)